tests/e2e/matmul/BUILD.bazel

```diff
@@ -542,7 +542,7 @@ iree_generated_e2e_runner_test(
         "requires-gpu-cdna3",
     ],
     target_backends_and_drivers = [
-        ("rocm", "rocm"),
+        ("rocm", "hip"),
     ],
```

Comment:
Thanks for updating these! FYI @erman-gurses I'd expect these tests to show up on the CI as part of https://github.com/iree-org/iree/issues/17159